src/lib/edge-functions/registry.ts

```diff
@@ -80,10 +80,11 @@ function traverseLocalDependencies(
 }
 
 export class EdgeFunctionsRegistry {
+  public importMapFromDeployConfig?: string
```

Comment:
I don't see this referenced any where else in this diff. Assuming it's a type addition, no runtime effect.

Reply:
I moved it from L86 to separate private properties from public ones.